src/app/dataset-view/additional-components/metadata-component/metadata.component.html

```diff
@@ -472,5 +471,13 @@
                 </ng-container>
             </div>
         </ng-container>
+        <ng-container
+            *ngIf="
+                !currentState?.metadata?.metadata?.currentSource &&
+                !currentState?.metadata?.metadata?.currentTransform
+            "
+        >
+            <h3 class="box-title text-center">There is no metadata.</h3>
```

Comment:
Please point to the same action as in Data tab - to start defining `SetPollingSource` for root or `SetTransform` for derived.